### PR TITLE
Revert "Add multiple colors"

### PR DESCRIPTION
**Pull request by mujibluth**
Reverts mujibluth/reviewing-a-pull-request#6

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>Javascript Tetris</title>
+  <title>Javascript Tetris by : mujibluth</title>
```
Comment (mujibluth) [inline index.html:4]: That's true

Comment (mujibluth) [inline index.html:4]: it's a good idea

**Review by mujibluth — COMMENTED** [review]
good idea

**Review by mujibluth — COMMENTED** [review]
good

File: index.html (hunk 1 above)
Comment (mujibluth) [inline index.html:4]: commented on specific lines

**Review by mujibluth — COMMENTED** [review]
good

**Review by abdul-mujib — APPROVED** [review]
good

**Pull request merged**